assets/js/src/common/subscribers_cache_message.tsx

```diff
@@ -43,6 +43,7 @@ export function SubscribersCacheMessage({ cacheCalculation }: Props): JSX.Elemen
         {MailPoet.I18n.t('recalculateNow')}
       </Button>
       <div className="mailpoet-gap" />
+      {errors.length > 0 && <Notice type="error">{errors.map((error) => <p key={error}>{error}</p>)}</Notice>}
```

Comment:
If you used `APIErrorsNotice` instead of regular `Notice` the code would be much more simple here.